NOnion/Directory/TorDirectory.fs

```diff
@@ -174,6 +175,9 @@ type TorDirectory =
                        Seq.filter(fun router ->
                            Seq.contains "Guard" router.Flags
                        )
+                   | Exit ->
+                       Seq.filter(fun router -> Seq.contains "Exit" router.Flags
```

Comment:
@aarani yeeesh, this is such a shitty formatting, can you file a bug in fantomless please

Reply:
Will do it as soon as I was able to cook up a minimal test case

Reply:
Ok, do you mind I don't merge until that happens, so that we don't forget about the issue. Thanks

Reply:
https://github.com/nblockchain/fantomless/issues/9